app/src/main/java/fr/free/nrw/commons/auth/SignupActivity.java

```diff
@@ -61,4 +63,13 @@ public void onBackPressed() {
             super.onBackPressed();
         }
     }
+
+    @Override
+    public void applyOverrideConfiguration(final Configuration overrideConfiguration) {
+        if (Build.VERSION.SDK_INT == 21 || Build.VERSION.SDK_INT == 25 && (
```

Comment:
25 is Nougat right? https://developer.android.com/reference/android/os/Build.VERSION_CODES#N_MR1
Did you mean 22 by any chance?

Reply:
I am extremely sorry, should have reviewed the code before pushing, this line intended to say for API levels between 21 to 25, have made the correction.